### ISSUE TITLE
Wrong redirect link at DuckDuckGo

### ISSUE THREAD

**Issue by OnSive** [MicrosoftDocs/windows-uwp#1929]
**TL;DR**
I was searching for the `ComboBoxItem` style and template but the link which is shown at DuckDuckGo does not redirect properly.

---

Search results:
https://duckduckgo.com/?q=combobox+item+style

The second item seems correct  with the right description
> ComboBoxItem styles and templates - msdn.microsoft.com

But the link https://msdn.microsoft.com/en-us/library/windows/apps/mt299115.aspx does redirect to the general XAML styles page which does not equals the description on DuckDuckGo.

---

**EDIT:**
Just noticed that the llink `ComboBoxItem styles and templates` on ComboBoxItem Class doesn't work either.
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.comboboxitem#see-also

Does the page still exist?

**Comment by OnSive** [comment]
PING @knicholasa

**Comment by hkhmsft** [comment]
It appears the links have been since fixed. Closing this Issue.
